routes.php

```diff
@@ -418,3 +418,141 @@
         echo json_encode(['success' => false, 'errorMessage' => 'Invalid action.']);
     }
 });
+
+//Add movie post route
+post($baseRoute.'movies/add', function() {
+    require_once 'src/controller/MovieController.php';
+    $movieController = new MovieController();
+    parse_str(file_get_contents("php://input"), $_POST); // Parse the PUT request

```

Comment:
think this is unnecessary on a POST request (to my best knowledge it's only necessary for PUT requests)